### ISSUE TITLE
bring on the primates

### ISSUE THREAD

**Issue by touretzkyds** [touretzkyds/KnowledgeGraphDemo#25]
I believe there are around 300 primate species. Since the addition of new taxa is now largely automated, it shouldn't be too hard to add all of them to our knowledge graph. Let's do that so we can experiment with more dense graphs.

For now we can limit ourselves to non-extinct primate species.

**Comment by deyaliao** [comment]
I downloaded the ITIS database associated with the order [Primates](https://www.itis.gov/servlet/SingleRpt/SingleRpt?search_topic=TSN&search_value=180089#null)), which happens to include all the species and subspecies that are under the respective order (and everything taxonomically ranked in between). The Primates page indicates 520 species (granted, many are invalid entries, so cut that number down a bit). The reptilia class happened to have 10467 species, which might be significantly more than we’re ready to experiment for, and I originally wanted to download the entire reptilia class so I didn’t have to manually download the 100+ reptiles that had type species. However, the data took too long to run (30+ minutes), and for some reason the query would time out at arbitrary intervals, so I just gave up. I ended up downloading the 3/4 orders under reptilia excluding Squamata (10000 species itself), so there’s that, too. At least there’s more type species information. 

Quick note: initially with the 100 species, I actually went down to the subspecies level (the ITIS database includes that information along with each species), but in the hopes of speeding things up a bit, I decided to forgo sub-species information––which honestly really was the initial plan. This is why although I added the primates + reptilia species, there still is around a similar number of taxa in the database (also because I deleted some of Arthur’s stuff)

Regardless, with more species in, we can see the taxa in our database:
```
PREFIX kgo: <http://solid.boltz.cs.cmu.edu:3030/ontology/>
PREFIX rdfs:  <http://www.w3.org/2000/01/rdf-schema#> 

  SELECT DISTINCT ?yLabel ?rLabel ?typeS  ?typeSof WHERE {
    ?a kgo:taxonName "Animalia"@en.
    ?y kgo:subTaxonOf* ?a.
    ?y kgo:taxonName ?yLabel.
  OPTIONAL{?y kgo:typeSpecies ?typeS.}
  OPTIONAL{?y kgo:typeSpeciesOf ?typeSof.}
  	?y kgo:taxonRank [rdfs:label ?rLabel] .  
  }
  ORDER BY ?yLabel
 
```
